profiler_detect_target_2024.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import multiprocessing as mp
+import pathlib
+import queue
+import time
+import numpy as np 
+import os
+import pandas as pd
+
+import cv2
+import yaml
+
+from modules.detect_target import detect_target_worker
+
+from modules.video_input import video_input_worker
+from utilities.workers import queue_proxy_wrapper
+from utilities.workers import worker_controller
+from utilities.workers import worker_manager
+
+
+
+CONFIG_FILE_PATH = pathlib.Path("config.yaml")
+
+
+def main() -> int:
+    """
+    copied from airside code main function
+    """
+    # Open config file
```

Comment:
Do you think you could explain what the difference between this file is and `profile_detect_target.py`? Just trying to understand what the difference is.